.buildkite/hooks/pre-exit

```diff
@@ -1,34 +0,0 @@
-#!/bin/bash
-
-set -eo pipefail
-
-if [[ "$BUILDKITE_STEP_KEY" == *"integration-tests"* ]]; then
-  if [[ -z "${WORKSPACE-""}" ]]; then
-      WORKSPACE=$(git rev-parse --show-toplevel)
-  fi
-  source "${WORKSPACE}/.buildkite/scripts/common.sh"
-
-  # Perform cleanup of integration tests resources
-  echo "--- Cleaning up integration test resources"
-  STACK_PROVISIONER=serverless SNAPSHOT=true mage integration:clean
-  SNAPSHOT=true mage integration:clean
-fi
-
-if [ -n "$GOOGLE_APPLICATION_CREDENTIALS" ]; then
-  if test -f "$GOOGLE_APPLICATION_CREDENTIALS"; then
-    rm $GOOGLE_APPLICATION_CREDENTIALS
-  fi
-fi
-
-if [ -n "$TEST_INTEG_AUTH_GCP_SERVICE_TOKEN_FILE" ]; then
-  if test -f "$TEST_INTEG_AUTH_GCP_SERVICE_TOKEN_FILE"; then
-    rm $TEST_INTEG_AUTH_GCP_SERVICE_TOKEN_FILE
-  fi
-fi
-
-source .buildkite/scripts/unset-secrets.sh
```

Comment:
This is what is caused the problem with the secret. Lets revert the file and remove this line only.

Because on like 13 we also perform clean up of the resources.